src/zeit/retresco/search.py

```diff
@@ -0,0 +1,40 @@
+import elasticsearch
+import json
+import zeit.cms.interfaces
+import zeit.cms.result
+import zeit.retresco.interfaces
+import zope.interface
+
+
+@zope.interface.implementer(zeit.retresco.interfaces.IElasticsearch)
```

Comment:
Sollen wir wirklich mit class-decorators anfangen? Ganz vivi verwendet das bisher nicht, und mit grok passt es AFAIK auch nicht recht zusammen. (In zeit.web wird es hingegen verwendet...) Trotzdem, ich glaub ich wäre dafür, das im vivi erstmal nicht anzufangen.